src/screens/Topics/index.js

```diff
@@ -12,6 +12,7 @@ import {
 } from 'react-native';
 import {useNavigation} from '@react-navigation/core';
 
+import {useSafeAreaInsets} from 'react-native-safe-area-context';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `useSafeAreaInsets` from 'react-native-safe-area-context' is a good addition. It will help to handle the safe area insets in a better way, especially for devices with notches or home indicators.
```diff
+ import {useSafeAreaInsets} from 'react-native-safe-area-context';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->